juriscraper/opinions/united_states/state/connappct.py

```diff
@@ -13,8 +15,6 @@
 class Site(conn.Site):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.crawl_date = date.today()
-        self.url = "http://www.jud.ct.gov/external/supapp/archiveAROap{year}.htm".format(
-            year=self.crawl_date.strftime("%y")
-        )
         self.court_id = self.__module__
+        current_year = date.today().strftime("%y")
```

Comment:
no need to duplicate this from the Supreme Court.

Reply:
Removed